### PR TITLE
Improving the project status section of our README

### PR DESCRIPTION
**Pull request by arschles**
**What is the problem I am trying to address?**

Athens is not really "beta" anymore because teams are using it and we are already taking backward compatibility into consideration actively.

**How is the fix applied?**

I've changed the language in the section that says the project is beta.

>Note: I believe that the version number is a different but related discussion that we should have.

I've also updated the contributing documentation and removed the section about dev meetings, because we don't have them anymore.

**Mention the issue number it fixes or add the details of the changes if it doesn't have a specific issue.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,24 +49,28 @@ Speaking of contributing, read on!
 
 # Contributing
 
-This project has plenty of interesting and challenging work to do and a wonderful community.
+If you use Athens for your development workflow, we hope that you'll consider contributing back to the project. Athens is widely used and has plenty of interesting work to do, from technical challenges to technical documentation to release management. We have a wonderful community that we would love you to be a part of. [Absolutely everyone is welcome](https://arschles.com/blog/absolutely-everybody-is-welcome/).
+
+The quickest way to get involved is by [filing issues](https://github.com/gomods/athens/issues/new/choose) if you find bugs or find that you need Athens to do something it doesn't.
+
+You also might consider contributing code back to the project. If you're familiar with the GitHub contribution process, you'll probably be familiar with our process. Generally, you may optionally find or submit an issue, and then submit a pull request (PR) to fix that issue. See [here](https://docs.gomods.io/contributing/) for some of the project-specific details.
```
Comment (twexler) [inline README.md]: The first sentence here is a bit repetitive with the first sentence of the first paragraph. Maybe something like:

> If you'd like to help us tackle the technical challenges, and are familiar with the GitHub contribution process, the Athens community welcomes your contribution!

File: README.md [hunk 2]
```diff
@@ -47,24 +49,28 @@ Speaking of contributing, read on!
...
+
+>If you're not familiar with the standard GitHub contribution process, which Athens mostly follows, please see [this section of our documentation](https://docs.gomods.io/contributing/new/) to learn how to contribute. You can also take advantage of [@bketelsen](https://github.com/bketelsen)'s [great video](https://www.youtube.com/watch?v=bgSDcTyysRc) on how to contribute code. The information in these documents and videos will help you not only with this project, but can also help you contribute to many other projects on GitHub.
 
-If you find a bug or want to fix a bug, we :heart: PRs and issues! If you see an issue
-in the [queue](https://github.com/gomods/athens/issues) that you'd like to work on, please just post a comment saying that you want to work on it. Something like "I want to work on this" is fine.
+If you decide to to contribute but aren't sure what to work on, we have a well maintained [list of good first issues](https://github.com/gomods/athens/contribute) that you should look at. If you find one that you would like to work on, please post a comment saying "I want to work on this", and then it's all yours to begin working on.
```
Comment (twexler) [inline README.md]: Didn't know about `/contribute`! Nice.

Comment (arschles) [reply README.md]: I learned about it a few weeks ago 🎉

**Comment by arschles** [comment]
Thanks @twexler - I updated according to your comment 💚

File: README.md [hunk 3]
```diff
@@ -47,24 +49,28 @@ Speaking of contributing, read on!
...
 
-If you decide to contribute (we hope you do :smile:), the process is familiar and easy if you've used Github before. There are no long documents to read or complex setup. If you haven't used Github before, the awesome [@bketelsen](https://github.com/bketelsen) has created a good overview on how to contribute code - see [here](https://www.youtube.com/watch?v=bgSDcTyysRc).
+>We do recommend that you choose one of the issues on the above list, but you may also consider a different issue from our [entire issue list](https://github.com/gomods/athens). Many of the issues on that list are more complex and challenging.
```
Comment (robjloranger) [inline README.md]: `/issues`

Comment (arschles) [reply README.md]: Good catch, fixed. Thanks @robjloranger

File: README.md [hunk 4]
```diff
@@ -47,24 +49,28 @@ Speaking of contributing, read on!
...
-in the [queue](https://github.com/gomods/athens/issues) that you'd like to work on, please just post a comment saying that you want to work on it. Something like "I want to work on this" is fine.
+If you decide to to contribute but aren't sure what to work on, we have a well maintained [list of good first issues](https://github.com/gomods/athens/contribute) that you should look at. If you find one that you would like to work on, please post a comment saying "I want to work on this", and then it's all yours to begin working on.
 
-If you decide to contribute (we hope you do :smile:), the process is familiar and easy if you've used Github before. There are no long documents to read or complex setup. If you haven't used Github before, the awesome [@bketelsen](https://github.com/bketelsen) has created a good overview on how to contribute code - see [here](https://www.youtube.com/watch?v=bgSDcTyysRc).
+>We do recommend that you choose one of the issues on the above list, but you may also consider a different one from our [entire list](https://github.com/gomods/athens). Many of the issues on that list are more complex and challenging.
```
Comment (robjloranger) [inline README.md]: The issues URL is https://github.com/gomods/athens/issues

Comment (arschles) [reply README.md]: @robjloranger good catch, thanks. Fixed!

**Comment by arschles** [comment]
I'm going to merge this since it's documentation and has 2 approvals

**Pull request merged**